### PR TITLE
Changes to glossary

### PR DESCRIPTION
**Pull request by pavpanchekha**
All minor, thank goodness. No other changes to end matter (skipped, changes, biblio)

### CODE REVIEW

File: book/glossary.md [hunk 1]
```diff
@@ -307,8 +307,8 @@ CSS properties (such as font sizing) are inherited by default.
 Style
 
 : All the pieces of information necessary to determine the visual
-display of an element. Also the name of a corresponding attribute to
-specify inline styles.
+display of an element. Also the name of the HTML element that
+specifies inline styles.
```
Comment (pavpanchekha) [inline book/glossary.md:311]: I guess it is also an attribute, actually, now that I think about this. So maybe ignore this change.

File: book/glossary.md [hunk 2]
```diff
@@ -165,7 +165,7 @@ text and the document object).
 Event
 
 : A way for JavaScript to observe that something has happened on the
-document, and customize its results.
+document and react.
```
Comment (chrishtr) [inline book/glossary.md:168]: I prefer the old text

File: book/glossary.md [hunk 3]
```diff
@@ -230,7 +230,7 @@ Networking
 Cookie
 
 : A piece of persistent, per-site state stored by web browsers
-to enable use cases like user logged-in status for access-controlled content.
+to enable use cases like user login for access-controlled content.
```
Comment (chrishtr) [inline book/glossary.md:233]: Not critical IMO

File: book/glossary.md (hunk 2 above)
Comment (pavpanchekha) [reply book/glossary.md]: Done,. kept the comma change.

**Pull request merged**